helper/metricsutil/wrapped_metrics.go

```diff
@@ -59,17 +63,30 @@ func (m *ClusterMetricSink) MeasureSinceWithLabels(key []string, start time.Time
 func BlackholeSink() *ClusterMetricSink {
 	sink, _ := metrics.New(metrics.DefaultConfig(""),
```

Comment:
Could we do `return NewClusterMetricSink("", &metrics.BlackholeSink{})` over here?

Reply:
Yeah, good point, I added the second factory later and then forgot to refactor this one.   I'll fix that and commit.

Reply:
On second thought, I'll commit now and fix later.